update_prevalence.py

```diff
@@ -324,8 +332,6 @@ class CanadaRegionalVaccinationReports(pydantic.BaseModel):
 
     class Report(pydantic.BaseModel):
         date_: date = pydantic.Field(alias="date")
-        total_cases: int
-        total_tests: Optional[int]
```

Comment:
I removed these because they're not used (we only use the vaccination-related items).  Validation started failing because `total_cases` started returning only null.  Perhaps `api.covid19tracker.ca` had the same issue with the breaking API changes from `api.opencovid.ca`.